_ont/ont-fs-com-mod.md

```diff
@@ -0,0 +1,7 @@
+---
+title: FS Modded Firmware for Huawei MA5671A and FS.com GPON-ONU-34-20BI
+has_children: false
+redirect_to: /ont-huawei-ma5671a-fs-mod
+layout: default
+parent: ONT FS.com
```

Comment:
```suggestion:
parent: ONT FS.com GPON ONU Stick with MAC (GPON-ONU-34-20BI)
```

and modify on `ont-fs-com-gpon-onu-stick-with-mac.md`:
`has_children: false` to `has_children: true`